shared/js/background/load.es6.js

```diff
@@ -80,7 +80,8 @@ function loadExtensionFile (params) {
                     xhr.data = returnResponse(xhr, params.returnType)
                     resolve(xhr)
                 } else if (xhr.status === 304) {
-                    reject(new Error(`${url} returned 304, resource not changed`))
+                    console.log(`${url} returned 304, resource not changed`)
```

Comment:
I updated this to a log instead of error and I'm returning an empty object with just a status code. I think it makes more sense to handle the 304 case explicitly.